saas/axops/src/ui/src/app/views/+settings/system-settings/spot-instances-panel/spot-instances-panel.component.ts

```diff
@@ -17,17 +18,16 @@ export class SpotInstancesPanelComponent implements OnInit {
 
     ngOnInit() {
         this.systemService.getSpotInstanceConfig().subscribe(res => {
-            this.spotInstancesEnabled = res.status === 'True';
+            this.spotInstancesOption = res.status;
```

Comment:
@wokeGit please use 'asgs' field instead of status. Field might take 'all', 'partial' or 'none' value. Currently there is a bug in API: argoproj#176

Please test PR again once bug is resolved.

Reply:
Good point! Fixed.